eth/filters/bench_test.go

```diff
@@ -144,9 +144,9 @@ func benchmarkBloomBits(b *testing.B, sectionSize uint64) {
 	db.Close()
 }
 
-var bloomBitsPrefix = []byte("bloomBits-")
-
+//nolint:unused
```

Comment:
what is unused?

Reply:
`clearBloomBits`. 
It's used within a test which is `Skip`ped, so technically it's not totally unused, but as far as the linter is concerned, it's not used.